### ISSUE TITLE
Add audio waveform display option

### ISSUE THREAD

**Issue by eharris** [ozmartian/vidcutter#236]
It would be much easier to find good cut points if there was also an audio track waveform display.

**Comment by rcbell** [comment]
I strongly agree.  I need to edit online presentations where the video primarily consists of slides, but the audio is verbal discussion.  That discussion is what I most need to edit, to remove long pauses, background noise, etc. I had to use kdenlive on another computer in conjunction with vidcutter so that I could access its audio wave to guide me.
